### PR TITLE
Add `install` prepare plugin using `install` into tmt try

### PR DESCRIPTION
**Pull request by falconizmi**
Pull Request Checklist

* [x] implement the feature
* [x] write the documentation
* [x] extend the test coverage
* [x] adjust plugin docstring
* [x] mention the version
* [x] include a release note

### CODE REVIEW

**Comment by lukaszachy** [comment]
Isn't it more natural to call that option install? As in `tmt try --install zsh fedora@container....`

File: docs/releases.rst [hunk 1]
```diff
@@ -84,6 +84,9 @@ the ``clean`` subcommand. Users can clean resources from the last
 run or from a run with a given id. Users can also choose to skip
 cleaning ``guests``, ``runs`` or ``images``.
 
+``tmt try`` now supports :ref:`/stories/cli/try/option/install` option
```
Comment (happz) [inline docs/releases.rst:87]: Needs to be moved to the right release.

File: stories/cli/try.fmf [hunk 2]
```diff
@@ -146,3 +146,18 @@ link:
...
           - implemented-by: /tmt/trying.py
           - implemented-by: /tmt/steps/prepare/feature.py
+
+    /install:
+        story:
+            As a user I want an easy way to install local rpm package on the guest.
+        description: |
+            Install local rpm package on the guest.
+
+            .. versionadded:: 1.35
```
Comment (happz) [inline stories/cli/try.fmf:156]: Needs to be updated just like the release note.

**Review by happz — CHANGES_REQUESTED** [review]

File: stories/cli/try.fmf (hunk 2 above)
Comment (falconizmi) [reply stories/cli/try.fmf]: Done

File: docs/releases.rst (hunk 1 above)
Comment (falconizmi) [reply docs/releases.rst]: Done

Comment (happz) [reply docs/releases.rst:104]: Is it? I see it in 1.35.0 release section, shouldn't it be in the upcoming one, 1.37.0?

Comment (falconizmi) [reply docs/releases.rst]: Hm, I believe that I changed that .. maybe due some rebase from other branch or change in format of release number it went back to 1.35 .. for now its fixed

File: docs/releases.rst [hunk 3]
```diff
@@ -31,6 +31,8 @@ execution. Use ``tmt run discover -vvv`` to see the details.
 
 ``tmt try`` now supports :ref:`/stories/cli/try/option/epel` option
 backed by :ref:`prepare/feature</plugins/prepare/feature>` plugin.
+``tmt try`` now supports :ref:`/stories/cli/try/option/install` option
```
Comment (happz) [inline docs/releases.rst:34]:
```suggestion

``tmt try`` now supports :ref:`/stories/cli/try/option/install` option
```

**Review by happz — APPROVED** [review]

File: tmt/cli.py [hunk 4]
```diff
@@ -1752,6 +1752,9 @@ def init(
 @option(
     "--epel", is_flag=True, default=False,
     help="Enable epel repository.")
+@option(
+    "--install", default=[], metavar="PACKAGE", multiple=True,
+    help="Install local rpm package on the guest.")
```
Comment (psss) [inline tmt/cli.py]: It's not `local` packages only, right? We can install packages from repositories as well. Plus, the `install` plugin supports non-rpm packages as well.

**Review by psss — APPROVED** [review]
Thanks for implementing this! Looks good. In 1b7d1f59 I've just adjusted the option description (we support non-local and non-rpm packages as well) and the release note a bit.

File: docs/releases.rst (hunk 1 above)
Comment (psss) [reply docs/releases.rst]: Seems to be covered.

File: tmt/cli.py (hunk 4 above)
Comment (psss) [reply tmt/cli.py]: Fixed in 1b7d1f59.

**Pull request merged**